### DISCUSSION TITLE
Primitive for input dropdown selection

### DISCUSSION THREAD

**Discussion by gunnartorfis** [radix-ui/primitives#1203] [Help] [answered]
I'm implementing a very similar behaviour that exists on Google. Searching populates suggestions beneath the input where you can use keyboard navigation to select an item, etc. Which primitive is feasible for this behaviour? I've tried Dropdown Menu without success since it's triggered by a button and not input's value. 

Btw, I really love Radix UI and am currently using a few of your lovely primitives.

**Comment by liaujianjie** [comment, marked as answer]
What you described seems to be a combobox (here an [example from Downshift](https://www.downshift-js.com/use-combobox))—and I don't think Radix has it yet. There was some [early discussion](https://github.com/radix-ui/primitives/discussions/375#discussioncomment-1903810) back in early January 2022.

**Reply by gunnartorfis** [reply]
It is indeed a combobox, I couldn't remember the name 😆. Thanks a lot for the super-quick reply. I'll use Downshift for now 👍
